bin/GameIndex.yaml

```diff
@@ -26,6 +26,26 @@
 # ---------------------------------------------
 # -- Game List
 # ---------------------------------------------
+3828BB66:
```

Comment:
You need to check for the game serial.

Reply:
It's a prototype, there probably isn't one, so it might not work with our DB structure very well..

Reply:
If the crc isn't changed then adding it in gsdx is probably fine, not a fan adding it to gamedb without a serial check tho.

Reply:
The serial is same as their final versions' serials. But when you use "Run ELF..." for their real ELFs (DISKINFO.BIN for Jak 3 prototypes, FIREWIRE.IRX for Jak II prototypes), the emulator doesn't see any serial. That is why I had to add the CRCs themselves.

Reply:
I realise this, but the problem with adding CRC's like this is they are not unique and may clash with other games and could apply fixes that break others games, and I honestly am not willing to add that risk to fix some prototypes when you can probably patch the ISO's SYSTEM.CNF files to run the correct ELF instead of the protected one.

Reply:
Yes, they tried to disguise it. What looks like a normal SCES/SCUS game ELF is actually the protection.

My problem is not the prototype protection. The problem is with the pnach system. Protection ELFs for prototypes of the same game have identical ELF files. So pcsx2 can't make difference between them, unless I run the real game ELF directly.

Well I had a feeling that these changes wouldn't be accepted. And it is quite understandable. Thank you for your time. Maybe in the future the pnach system will be changed somehow...

No it is not really needed, we will just add both these commits into our branches.

Reply:
As I say, I'm happy for us to add the GSdx CRC's but I'm just not overly happy with the GameDB Stuff.

So feel free to modify your PR to just have the GSdx stuff :)